aea/multiplexer.py

```diff
@@ -341,14 +344,13 @@ async def _connect_all(self) -> None:
                 await self._connect_one(connection_id)
                 connected.append(connection_id)
             except Exception as e:  # pylint: disable=broad-except
-                self.logger.error(
-                    "Error while connecting {}: {}".format(
-                        str(type(connection)), str(e)
+                if not isinstance(e, (asyncio.CancelledError, CancelledError)):
+                    self.logger.exception(
+                        "Error while connecting {}: {}".format(
+                            str(type(connection)), repr(e)
+                        )
                     )
-                )
-                for c in connected:
-                    await self._disconnect_one(c)
-                break
```

Comment:
Where is the disconnection now taking place?

Reply:
in multiplexer.connect

Reply:
makes more sense, thanks